src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/PostgresDirectoryDAO.scala

```diff
@@ -154,7 +154,67 @@ class PostgresDirectoryDAO(protected val dbRef: DbReference,
     */
   override def removeGroupMember(groupId: WorkbenchGroupIdentity, removeMember: WorkbenchSubject): IO[Boolean] = ???
 
-  override def isGroupMember(groupId: WorkbenchGroupIdentity, member: WorkbenchSubject): IO[Boolean] = ???
+  override def isGroupMember(groupId: WorkbenchGroupIdentity, member: WorkbenchSubject): IO[Boolean] = {
+    val gm = GroupMemberTable.syntax("gm")
+
+    val memberClause: SQLSyntax = member match {
+      case WorkbenchGroupName(groupName) =>
+        val g = GroupTable.syntax("g")
+        sqls"sg.member_group_id = (select ${g.id} from ${GroupTable.as(g)} where ${g.name} = $groupName)"
+
+      case FullyQualifiedPolicyId(FullyQualifiedResourceId(resourceTypeName, resourceId), accessPolicyName) =>
+        val rt = ResourceTypeTable.syntax("rt")
+        val r = ResourceTable.syntax("r")
+        val p = PolicyTable.syntax("p")
+        sqls"""sg.member_group_id = (select ${p.groupId}
+              from ${ResourceTypeTable.as(rt)}
+              join ${ResourceTable.as(r)} on ${rt.id} = ${r.resourceTypeId}
+              join ${PolicyTable.as(p)} on ${r.id} = ${p.resourceId}
+              where ${rt.resourceTypeName} = $resourceTypeName)
+              and ${r.name} = $resourceId
+              and ${p.name} = $accessPolicyName)"""
+
+      case WorkbenchUserId(userId) => sqls"sg.member_user_id = $userId"
```

Comment:
Consider extracting `sqls"sg.member_user_id = "`?  I like it the way it is for readability, but for maintainability it might be better to extract it, especially if we end up just calling the sub query helper methods.  Either way is fine with me, but my preference would be to extract it, especially since it is a table alias/column that we are defining ourselves without scalike's help.